From 106b87b24a608fe1ea16c56bfe3caa72b3e8171b Mon Sep 17 00:00:00 2001
From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Wed, 21 Aug 2019 18:53:38 +0200
Subject: [PATCH] modules: update Gluon packages

d4c87ce6b06e simple-tc: depend on KERNEL_NET_ACT_POLICE or kmod-sched
73b301911d2d simple-tc: fix confusing program names in error messages
---
 modules | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/modules b/modules
index 047ea518..314375f2 100644
--- a/modules
+++ b/modules
@@ -17,4 +17,5 @@ PACKAGES_LUCI_BRANCH=openwrt-18.06
 PACKAGES_LUCI_COMMIT=4ba85e3d82b684262c570e38a72d2dc3bb712a13
 
 PACKAGES_GLUON_REPO=https://github.com/freifunk-gluon/packages.git
-PACKAGES_GLUON_COMMIT=524ae77e9bb20002252af19b63e6607155b3af4d
+PACKAGES_GLUON_BRANCH=v2018.2.x
+PACKAGES_GLUON_COMMIT=d4c87ce6b06ea280a08e7edc78050d053a3c0809
-- 
2.17.1

